### PR TITLE
Csi 1268 bug fix limit characters for near account

### PR DESCRIPTION
**Pull request by TejasvCypherock**

### CODE REVIEW

**Comment by irshadCypherock** [comment]
Task linked: [CSI-1268 BugFix: Limit characters for near account id](https://app.clickup.com/t/37308523/CSI-1268)

File: cysync/src/pages/mainApp/sidebar/wallet/addAccount/formStepComponents/Recipient.tsx [hunk 1]
```diff
@@ -304,7 +304,8 @@ const Recipient: React.FC<StepComponentProps> = props => {
   const checkNearAccount = async (address: string) => {
     const coin = COINS[coinDetails.slug];
     if (coin instanceof NearCoinData) {
-      if (address.includes('.')) return 'This is not a valid Near address';
+      if (address.includes('.') || (address + nearSuffix).length > 64)
+        return 'This is not a valid Near address';
```
Comment (irshadCypherock) [inline cysync/src/pages/mainApp/sidebar/wallet/addAccount/formStepComponents/Recipient.tsx:308]: Can we write the exact problem in the error msg? Something along the lines of `It should be less than 64 chars` or `It should end in .testnet`

File: cysync/src/store/hooks/flows/useSendTransaction.ts [hunk 2]
```diff
@@ -89,9 +89,13 @@ export const broadcastTxn = async (
         network: coin.network
       })
       .request();
+
     if (resp.status === 0) {
       throw new Error('brodcast-failed');
     }
+    if (typeof resp.data?.status?.SuccessValue === typeof undefined)
```
Comment (irshadCypherock) [inline cysync/src/store/hooks/flows/useSendTransaction.ts:96]: Can't we just use `resp.data?.status?.SuccessValue === undefined`

**Review by irshadCypherock — CHANGES_REQUESTED** [review]

File: cysync/src/pages/mainApp/sidebar/wallet/addAccount/formStepComponents/Recipient.tsx (hunk 1 above)
Comment (TejasvCypherock) [reply cysync/src/pages/mainApp/sidebar/wallet/addAccount/formStepComponents/Recipient.tsx:308]: Sure

**Review by irshadCypherock — APPROVED** [review]

**Pull request merged**